platform/android/detect.py

```diff
@@ -298,6 +298,17 @@ def mySpawn(sh, escape, cmd, args, env):
     env.Append(CPPFLAGS=['-DANDROID_ENABLED', '-DUNIX_ENABLED', '-DNO_FCNTL'])
     env.Append(LIBS=['OpenSLES', 'EGL', 'GLESv3', 'android', 'log', 'z', 'dl'])
 
+    ## arcore, should see about moving this into our module
```

Comment:
Same with this, need to see if we can move it into modules\arcore somehow, not sure if its possible

Reply:
Fixed in BastiaanOlij/godot#9.